src/components/resources/Resources.js

```diff
@@ -0,0 +1,123 @@
+    }, []);
+
+    const loaduserSearchByEmail = (email) => {
+        callApiGetuserSearchByEmail(email)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+                //console.log(parsed[0].id);
+                setUserID(parsed[0].id);
+            });
+    }
+
+    const callApiGetuserSearchByEmail = async (email) => {
+        const url = `${REACT_APP_API_ENDPOINT}/userSearchByEmail`;
+        //console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                email: email
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    useEffect(() => {
+        loadGetResources();
+    }, []);
+
+    const loadGetResources = async () => {
+        try {
+            const res = await callApiGetResources();
+            const parsed = JSON.parse(res.express);
+            setResources(parsed);
+
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    const callApiGetResources = async () => {
+
+        const url = `${REACT_APP_API_ENDPOINT}/getResources`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json",
+            }
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    return (
+        <div id="body">
+
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 110, left: '50%', transform: 'translate(-50%, -50%)' }}>
+                <Typography
+                    variant="h4"
+                    gutterBottom
+                    component="div">
+                    Resources
+                </Typography>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
+                {resources.map((resources) => (
+                    <Card style={{ width: '800px', marginBottom: '20px' }}>
+                        <CardContent>
+                            <Typography variant="h5" component="div">
+                                {resources.resourcesTitle}<br />
+                            </Typography>
+                            <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                Posted on {new Date(new Date(resources.dateTime).getTime() - (5 * 60 * 60 * 1000)).toLocaleDateString()}<br />
+                                by {resources.creatorName}<br />
+                            </Typography>
+                            <a href={`${resources.resourcesLink}`} target = "_blank">
```

Comment:
when clicking on a link that doesn't start with "https://", the user is redirected to a broken link. Ex: clicking on the link "www.instagram.com/eshah17" redirects to http://localhost:3000/www.instagram.com/eshah17

Additionally, there is the concern that the user doesn't input a valid url for a resource. Ex: the user inputs "test" as the url. They're directed to "http://localhost:3000/test"

I recommend going back to the create resource page and validating that the url the user inputs begins with "https://". This should resolve the issues above.